---kind:   - Troubleshootingproducts:    - Alauda Container Platform   - Alauda DevOps   - Alauda AI   - Alauda Application Services   - Alauda Service Mesh   - Alauda Developer PortalProductsVersion:   - 4.1.0,4.2.x---<!-- A type of document that involves encountering a fault, diag...it, performing root cause analysis, and providing solutions. --># libovsdb/cache: during libovsdb cache populate2 err=cache inconsistentPod 无法访问集群外网络 kube-ovn-controller 日志报错 'failed to get port group ovn.default.*: object not found' libovsdb 日志出现 'cache inconsistent: row with uuid * does not exist' 错误## Cause- libovsdb 缓存与 OVN_Northbound 数据库出现不一致- NB 数据库集群存在频繁切换现象## Resolution- kubectl -n kube-system rollout restart deploy kube-ovn-controller- 应用代码修复：https://github.com/kubeovn/kube-ovn/pull/2627## [workaround]- 重启 kube-ovn-controller 重建缓存## [Related Information]- Environment: CNI: Kube-OVN v1.9- 6641- ovn-nbctl- kube-ovn-controller- port_group- ovn_Northbound 数据库- Component: kube-ovn- Page ID: 144901987- Original Title: libovsdb/cache: during libovsdb cache populate2 err=cache inconsistent